projects/plugins/jetpack/sal/class.json-api-post-base.php

```diff
@@ -88,52 +222,79 @@ public function get_terms() {
 		return (object) $terms;
 	}
 
+	/**
+	 * Returns an object which holds the posts tag details
+	 *
+	 * @return object
+	 */
 	public function get_tags() {
-		$tags = array();
+		$tags  = array();
 		$terms = wp_get_post_tags( $this->post->ID );
 		foreach ( $terms as $term ) {
-			if ( !empty( $term->name ) ) {
-				$tags[$term->name] = $this->format_taxonomy( $term, 'post_tag', 'display' );
+			if ( ! empty( $term->name ) ) {
+				$tags[ $term->name ] = $this->format_taxonomy( $term, 'post_tag', 'display' );
 			}
 		}
 		return (object) $tags;
 	}
 
+	/**
+	 * Returns an object which holds the posts category details
+	 *
+	 * @return object
+	 */
 	public function get_categories() {
 		$categories = array();
-		$terms = wp_get_object_terms( $this->post->ID, 'category', array( 'fields' => 'all' ) );
+		$terms      = wp_get_object_terms( $this->post->ID, 'category', array( 'fields' => 'all' ) );
 		foreach ( $terms as $term ) {
-			if ( !empty( $term->name ) ) {
-				$categories[$term->name] = $this->format_taxonomy( $term, 'category', 'display' );
+			if ( ! empty( $term->name ) ) {
+				$categories[ $term->name ] = $this->format_taxonomy( $term, 'category', 'display' );
 			}
 		}
 		return (object) $categories;
 	}
 
+	/**
+	 * Returns an array of objects which hold the posts attachment information and numbers representing how many associated posts are found.
+	 *
+	 * @return array
+	 */
 	public function get_attachments_and_count() {
-		$attachments = array();
-		$_attachments = new WP_Query( array( 'post_parent' => $this->post->ID, 'post_status' => 'inherit', 'post_type' => 'attachment', 'posts_per_page' => '20' ) );
+		$attachments  = array();
+		$_attachments = new WP_Query(
+			array(
+				'post_parent'    => $this->post->ID,
+				'post_status   ' => 'inherit',
```

Comment:
Spaces somehow got inserted in the wrong place here.
```suggestion
				'post_status'    => 'inherit',
```

Reply:
Thank you!